### DISCUSSION TITLE
Build Cache will not be stored

### DISCUSSION THREAD

**Discussion by bonesoul** [vercel/vercel#9187] [Help]
getting this after my builds:

`Since the Node.js version () or package manager  could not be determined, Build Cache will not be stored`

Basically it's a next.js project living on vercel and the only thing is that my project exists in frontend/ folder (as I do have some other related non next.js libraries and helper tools in the same repo). In vercel project settings i do set the frontend/ as my root folder.

Any ideas?

**Comment by amyegan** [comment]
Thanks for reaching out @bonesoul. It looks like you are already working with someone on the support ticket you submitted, but I want to share the suggested workaround here in case anyone else is running into this problem. This may be related to Node 18 and the temporary solution is to [downgrade to Node 16 in your project settings](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/node-js#node.js-version).

**Reply by bonesoul** [reply]
Well I use a custom build command over vercel and I suspect that's the case.

Any ETA when node 18 support will be fixed?
